site/src/sources/materialUI.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "../schemas/design-system.schema.json",
+  "lastUpdated": "2023-04-24",
+  "name": "Material UI",
+  "description": "Material UI is a library of React UI components that implements Google's Material Design.",
+  "url": "https://mui.com/material-ui/getting-started/overview/",
+  "by": "MUI",
+  "components": [
+    {
+      "name": "Accordion",
```

Comment:
Do we typically just import the components we're interested in? Or is there a way to import links to all of the Material and Tailwind components? I'm only saying this if it's easy. If each component takes work to figure out, then we can do it piecemeal like this.